packages/graphql/src/schema/create-relationship-fields/create-relationship-fields.ts

```diff
@@ -255,7 +257,7 @@ function createRelationshipFields({
             }
         }
 
-        if (nestedOperations.has(RelationshipNestedOperationsOption.CREATE)) {
+        if (nestedOperations.has(RelationshipNestedOperationsOption.CREATE) && rel.settableOptions.onCreate) {
```

Comment:
I can see from the schema tests that `@settable` is also removing nested operations which are only supposed to be controlled by the `@relationship(nestedOperations: [])`. I imagine these are the lines that are causing this

```suggestion
        if (nestedOperations.has(RelationshipNestedOperationsOption.CREATE)) {
```